schedule_local.py

```diff
@@ -123,9 +123,9 @@ def execute_workflows(self) -> None:
       executor.map(self.execute_workflow, self.get_parent_workflow_directories())
 
   def get_project_from_service(self, service_path) -> str:
-    '''Gets the project id from the service JSON file.'''
+    '''Gets the project id from the service json file.'''
     with open(service_path) as service_file:
-      return json.load(service_file)['project_id']
+      return yaml.load(service_file)['project_id']
```

Comment:
Service credentials are saved as JSON from the UI.

Reply:
Because YAML is a superset of JSON this works, but JSON is moderately faster so switching back is a good call.
```suggestion
      return json.loads(service_file)['project_id']
```